engine.py

```diff
@@ -176,6 +188,27 @@ def _get_dialog_parent(self):
 
         return window
 
+    def _hide_menu_in_scripts(self):
+        """
+        Remove the entry in the VRED Scripts window
+        """
+        main_window = self._get_dialog_parent()
+        menu_names = main_window.menuBar().actions()
+
+        # Make it invisible for both Pro & Design
+        for menu_name in menu_names:
+            if menu_name.text() == "Scripts":
+                scripts_menu_items = menu_name.menu().actions()
+                for scripts_menu_item in scripts_menu_items:
+                    if scripts_menu_item.text() == "Shotgun":
+                        scripts_menu_item.setVisible(False)
+
+        # Check if it is Design and also "mute" the menu we made
+        if os.getenv("TK_VRED_EXECPATH").endswith("VREDDesign.exe"):
+            for menu_name in menu_names:
+                if menu_name.text() == "Scripts":
+                    menu_name.setVisible(False)
```

Comment:
With the suggestion above to find the `"Scripts"` action, we can avoid iterating to find the action menu a second time by now just calling:
```
if scripts_action:
    # find Shotgun menu and hide

    if os.getenv("TK_VRED_EXECPATH").endswith("VREDDesign.exe"):
        scripts_action.setVisible(False)